proposals/Fleetctl-trigger.md

```diff
@@ -0,0 +1,84 @@
+# Fleetctl trigger
+
+## Goal
+
+As a user, I would like to trigger a set of async jobs using `fleetctl`. For example, I’d like to
+trigger a vuln scan, or an MDM dep sync. 
+
+The proposed solution to accomplish this goal enables a new CLI command: 
+`fleetctl trigger --name <NAME>`.   
```

Comment:
We would need a `fleetctl trigger --list` I suppose.

Reply:
Or `fleetctl trigger help` or `fleetctl trigger<ENTER>` shows the possible names for `--name`? (we could hard-code them, and add them as we see fit, this is to not implement a list triggers API...)